patch/gem5.patch

```diff
@@ -15,10 +15,16 @@ index 0e266a6f4..1a2bb4174 100644
 +                "/opt/rocm/hip/lib",
 +                "/opt/rocm/hsa/lib",
 +                "/opt/rocm/libhsakmt/lib",
-+                "/usr/lib/x86_64-linux-gnu"
++                "/usr/lib/x86_64-linux-gnu",
++                "/opt/rocm/miopen/lib",
++                "/opt/rocm/miopengemm/lib",
++                "/opt/rocm/hipblas/lib",
++                "/opt/rocm/rocblas/lib"
             ]),
++           "HOME=/",
             "HSA_ENABLE_INTERRUPT=0"]
```

Comment:
there are some extra flags we may need declared in here, but if they are not necessary at this point, then no need to add now.